syntax = "proto3";

package mc.comm;

option go_package = "mc/comm";

message PBDirSpec {
    string base_dir = 1;
    repeated string image_list = 2;
    repeated string bin_buffer_list = 3;
    repeated string shader_code_list = 4;
    repeated string art_logic_list = 5;
    repeated string act_logic_list = 6;
    repeated string texture_list = 7;
    repeated string model_list = 8;
    repeated string shader_program_list = 9;
    repeated string material_list = 10;
    repeated string scene_list = 11;
    repeated string node_list = 12;
    repeated string light_list = 13;
}
